contracts/components/mina_base_split_gen.sol

```diff
@@ -51,23 +52,24 @@ library mina_base_split_gen {
     uint256 constant CONSTANTS_N = 1;
 
 
-    uint256 constant PERMUTATION_COLUMNS = 7;
+/*    uint256 constant PERMUTATION_COLUMNS = 7;
```

Comment:
If we're going to use params as a separate structure, then we can just delete the old one, I think.